cards.js

```diff
@@ -17,7 +17,28 @@ function shuffle(array){
 }
 
 function buildDeck(){
+  for(var i = 0; i < values.length; i++){ [i]
+    for(y in suits){ suits[y]
+    deck.push([i, suits[y]])
+    }
+  }
+  return(deck);
+}
+buildDeck();
+shuffle(deck);
 
-// Put some code in here!
-
+function highCard(){
+  player1 = deck.pop();
+  player2 = deck.pop();
+  if (player1[0] > player2[0]){
```

Comment:
Will this handle "Q" vs 3?

Instead of comparing the values directly, you should compare the index of those values in the `values` array:

```js
if( values.indexOf( player1[0] ) > values.indexOf( player2[0] ) )
```